app/graph/root_schema.rb

```diff
@@ -13,5 +13,8 @@
     mutation Types::MutationType
 
     instrument(:field, Util::AuthorizationInstrumentation.new)
-    max_depth 5
+
+    # FIXME: Determine why this was added, as the root submission request returns
+    # a depth of 13 (which, either way, seems incorrect).
+    # max_depth 5
```

Comment:
When the GraphiQL app boots at `http://localhost:5000/graphiql` this query is immediately (and automatically) executed, leading to the unbootability of the explorer. When used as an API I suspect this is a non-issue, but we'll need to track down the why's here:

```graphql
     query IntrospectionQuery {
        __schema {
          queryType { name }
          mutationType { name }
          subscriptionType { name }
          types {
            ...FullType
          }
          directives {
            name
            description
            locations
            args {
              ...InputValue
            }
          }
        }
      }

      fragment FullType on __Type {
        kind
        name
        description
        fields(includeDeprecated: true) {
          name
          description
          args {
            ...InputValue
          }
          type {
            ...TypeRef
          }
          isDeprecated
          deprecationReason
        }
        inputFields {
          ...InputValue
        }
        interfaces {
          ...TypeRef
        }
        enumValues(includeDeprecated: true) {
          name
          description
          isDeprecated
          deprecationReason
        }
        possibleTypes {
          ...TypeRef
        }
      }

      fragment InputValue on __InputValue {
        name
        description
        type { ...TypeRef }
        defaultValue
      }

      fragment TypeRef on __Type {
        kind
        name
        ofType {
          kind
          name
          ofType {
            kind
            name
            ofType {
              kind
              name
              ofType {
                kind
                name
                ofType {
                  kind
                  name
                  ofType {
                    kind
                    name
                    ofType {
                      kind
                      name
                    }
                  }
                }
              }
            }
          }
        }
      }
```